scripts/up-test-cluster.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+function up-test-cluster() {
+    echo "Creating cluster $CLUSTER_NAME"
+    CLUSTER_YAML_PATH=$TEST_DIR/$CLUSTER_NAME.yaml
+    SSH_KEY_PATH=$TEST_DIR/id_rsa
+    ssh-keygen -P cni-test -f $SSH_KEY_PATH
+
+
+    $TESTER_PATH eks create config --path /tmp/aws-k8s-tester-eks.yaml
```

Comment:
Recommend putting `/tmp/aws-k8s-tester-eks.yaml` inside `$TEST_DIR` so you can test different clusters simultaneously.

Reply:
I added a separate test directory per cluster, so that both "1 cluster per test" and "1 cluster multiple tests" are supported.